.env.template

```diff
@@ -143,6 +143,13 @@ OPENAI_API_KEY=your-openai-api-key
 ### WEB BROWSING
 ################################################################################
 
+### Chrome WebDriver config
```

Comment:
Move this below browsing as most users should select a browser before configuring this

Reply:
should be resolved by 6491788 .